services/src/main/java/org/broadleafcommerce/payment/service/gateway/DefaultPayPalCheckoutTransactionService.java

```diff
@@ -347,7 +357,8 @@ protected Capture capturePayment(Authorization auth, PaymentRequest paymentReque
         capture.setIsFinalCapture(true);
         Amount amount = new Amount();
         amount.setCurrency(paymentRequest.getOrderSubtotal().getCurrency().getCurrencyCode());
-        amount.setTotal(Objects.toString(paymentRequest.getTransactionTotal(), null));
+
+        amount.setTotal(getStringAmount(paymentRequest.getTransactionTotal()));
         capture.setAmount(amount);
```

Comment:
For the `Capture` object, is there a way to pass the `PaymentRequest#transactionReferenceId`?

Reply:
I don't see anything I can use to pass the `PaymentRequest#transactionReferenceId`

Reply:
I found only `Capture#invoiceNumber`. Do you think we can use it as `transactionReferenceId`?

Reply:
I think their intention with this field is to pass something like the `Payment#id`, but I think we should pass the `transactionReferenceId`.